core/src/main/kotlin/org/evomaster/core/problem/rest/service/AbstractRestFitness.kt

```diff
@@ -808,7 +796,14 @@ abstract class AbstractRestFitness : HttpWsFitness<RestIndividual>() {
             dto.additionalInfoList
         )
 
-        handleExternalServiceInfo(individual, fv, dto.additionalInfoList)
+        val wmStarted = handleExternalServiceInfo(individual, fv, dto.additionalInfoList)
+        if(wmStarted){
+            /*
+                Quite tricky... if a WM instance was started as part of this test case evaluation,
+                then the results are invalid, as they are based on WM not being there.
+             */
+            return null
```

Comment:
is this still valid? 
in last meeting, didn't we write `We have on purpose a IP address that is left empty (nothing running on it) to deal with such case.
So, when we have an host lookup in SUT (with no port), defaulting to resolve to that in Hostname Action
should solve such issue, isn't it?
So, no need to kill the test. Actually, make sure we can cover those cases (eg, we have a E2E in which explicitely
check it is handle by not covering 400 status in a branch)`
but maybe best to do it in a different branch. i ll create a Trello card for it

Reply:
i guess maybe we could just comment out `null` here, as anyway this behavior is already disabled in the `Archive`

Reply:
If we don't disable in the `Archive`, we have to return `null` at this point to avoid tests without WM setup in the generated tests.